daemons/gptp/common/ptp_message.cpp

```diff
@@ -960,6 +960,13 @@ void PTPMessageFollowUp::processMessage(IEEE1588Port * port)
 		goto done;
 	}
 
+	if (sync->getTimestamp()._version != port->getTimestampVersion())
+	{
+		GPTP_LOG_ERROR
```

Comment:
We probably should check that the versions match, but this seems like an odd error message to use for mismatched version fields.